tools/flang2/flang2exe/lldebug.cpp

```diff
@@ -1847,8 +1847,12 @@ lldbg_emit_parameter_list(LL_DebugInfo *db, DTYPE dtype, DTYPE ret_dtype,
           lldbg_emit_modified_type(db, ret_dtype, SPTR_NULL, findex);
     else
 #endif
-      retval_mdnode =
-          lldbg_emit_type(db, ret_dtype, SPTR_NULL, findex, true, false, true);
+      if (DT_ISBASIC(ret_dtype))
+        retval_mdnode =
+         lldbg_emit_type(db, ret_dtype, SPTR_NULL, findex, false, false, true);
+      else
+        retval_mdnode =
+         lldbg_emit_type(db, ret_dtype, SPTR_NULL, findex, true, false, true);
```

Comment:
Recommend the following style since there is if-else branch if `CUDA_DEVICE` is defined.

```
      retval_mdnode = lldbg_emit_type(db, ret_dtype, SPTR_NULL, findex,
          DT_ISBASIC(ret_dtype) ? false : true, false, true);
```

Reply:
@bhuvanendrakumarn do you have a comment or an update here?